railties/lib/rails/generators/rails/encrypted_secrets/templates/config/secrets.yml.enc

```diff
@@ -0,0 +1,2 @@
+production:
+  secret_key_base: <%= SecureRandom.hex(64) %>
```

Comment:
This is a generator template; the randomness occurs once when the file's created, not each time it's read.